### PR TITLE
Port send-nsca to python3

### PR DESCRIPTION
**Pull request by asottile**

### CODE REVIEW

**Comment by asottile** [comment]
@tomelm seems we don't have travis-ci set up for this repository
@talyian @Roguelazer would love a revew :)

travis-ci passing here: https://travis-ci.org/asottile/send_nsca

File: send_nsca/__init__.py [hunk 1]
```diff
@@ -25,7 +25,8 @@ def send_nsca(status, host_name, service_name, text_output, remote_host, **kwarg
         n = NscaSender(remote_host=remote_host, **kwargs)
         n.send_service(host_name, service_name, status, text_output)
         n.disconnect()
-    except Exception, e:
+    except Exception as e:
+        import traceback; traceback.print_exc()
```
Comment (talyian) [inline send_nsca/__init__.py]: is this intentional?

Comment (asottile) [inline send_nsca/__init__.py]: I could remove it, but it makes this debuggable at all on failure

**Comment by talyian** [comment]
lg2m

**Pull request merged**